linear_algebra/src/schur_complement.py

```diff
@@ -31,16 +31,18 @@ def schur_complement(
     shape_c = np.shape(mat_c)
 
     if shape_a[0] != shape_b[0]:
-        raise ValueError(
-            f"Expected the same number of rows for A and B. \
-            Instead found A of size {shape_a} and B of size {shape_b}"
+        msg = (
+            "Expected the same number of rows for A and B.             "
+            f"Instead found A of size {shape_a} and B of size {shape_b}"
```

Comment:
```suggestion
        msg = (
            "Expected the same number of rows for A and B. "
            f"Instead found A of size {shape_a} and B of size {shape_b}"
```
Same as above but here it's the trailing `\` which is affecting it.